content/about/security-release-process/security-release-process.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Security Release Process
+description: Release process for Istio security vulnerabilities.
+weight: 35
+page_icon: /img/security-release.svg
+---
+
+The Istio team follows the process below when releasing patches for critical
+security vulnerabilities.
+
+## Security vulnerability disclosures
+
+Please refer to the [disclosure process](/about/security-vulnerabilities/).
+
+## Istio security team
+
+Security vulnerabilities should be handled quickly and privately.
+The primary goal of this process is to reduce the total time customers are
+vulnerable to publicly known exploits.
+
+The Istio security team is responsible for organizing the entire response
+including internal communication and external disclosure. The team enlists help
+from relevant developers and release leads to successfully run this process.
+
+The Istio security team consists of volunteers subscribed to the
+private [`istio-security-vulnerabilities@google.com`](mailto:istio-security-vulnerabilities@google.com)
+list.
+
+### Istio security team membership
+
+Membership to the Istio security team is based on the roles defined below.
+
+#### Roles
+
+* Fix lead
+
+Per issue, Fix Lead sees the issue through to the end.
+
+* Release lead
+
+Per issue, Release Lead is in charge of releasing the fix.
+
+* Disclosure lead
+
+Handles public messaging around the bug, and the documentation of how to upgrade.
+Explains the severity of the vulnerability and requests CVEs (Common Vulnerability Exposures).
+
+* Triage
+
+Makes sure the relevant people are notified, also responds to issues
```

Comment:
To fix consistency and grammar, I suggest:
* **Triage lead:** The triage lead notifies the relevant people and responds to issues other than security vulnerability issues. This person is the escalation path for a vulnerability.

Keep in mind that the original text is not clear regarding the escalation path. I edited the sentence to correct the grammar but the message is not very clear.